packages/commons/src/utils/getAll.ts

```diff
@@ -14,3 +14,30 @@ export async function getAllFromPaginated<A>(
 
   return await getAllFromOffset(0);
 }
+
+export const createPollingByCondition =
+  <T>(fetch: () => Promise<T>) =>
```

Comment:
I would create a dedicated utile file for this or maybe rename this file in something more meaningful like apiUtils